packages/server/tests/acceptance/htsPrecompile/tokenCreate.spec.ts

```diff
@@ -697,3 +701,15 @@ describe('@tokencreate HTS Precompile Token Create Acceptance Tests', async func
     });
   });
 });
+
+async function associateToken(mainContractOwner: any, mainContractAddress: any, HTSTokenContractAddress: any, TX_SUCCESS_CODE: number, mainContractReceiverWalletFirst: any, accounts: AliasAccount[], mainContractReceiverWalletSecond: any) {
```

Comment:
This method does a few things and it's not clear why.
It would be cleaner to have it do a single operation and verify, then have 3 calls feeding in the appropriate params

```ts
async function associateTokenAndVerifyEvent(contract: any, entityToAssociate: string, tokenAddress: string, expectedTxResponseCode) {
  const tx = await contract.associateTokenPublic(entityToAssociate, tokenAddress, { gasLimit: 10000000 });
  expect((await tx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode).to.equal(expectedTxResponseCode);
}
```

then call
```ts
await associateTokenAndVerifyEvent(mainContractReceiverWalletFirst, mainContractAddress, HTSTokenContractAddress, TX_SUCCESS_CODE);

await associateTokenAndVerifyEvent(mainContractOwner, accounts[1].wallet.address, HTSTokenContractAddress, TX_SUCCESS_CODE);

await associateTokenAndVerifyEvent(mainContractOwner, accounts[2].wallet.address, HTSTokenContractAddress, TX_SUCCESS_CODE);
```

Reply:
Done.